DynamicTablesPkg/DynamicTables.dsc.inc

```diff
@@ -30,7 +30,7 @@
 [LibraryClasses.AARCH64]
   DynamicTablesScmiInfoLib|DynamicTablesPkg/Library/DynamicTablesScmiInfoLib/DynamicTablesScmiInfoLib.inf
 
-[Components.common]
+[Components.ARM, Components.AARCH64, Components.X64]

```

Comment:
Is it possible to let `[Components.common]` ?
I assume the CI will only build the supported architectures, i.e. not `IA32` anymore